docs/en-US/component/tag.md

```diff
@@ -73,45 +73,50 @@ tag/checkable
+| type                | type of Tag                          | ^[enum]`'success' \| 'info' \| 'warning' \| 'danger'`    | —       |
+| closable            | whether Tag can be removed           | ^[boolean]                                               | false   |
+| disable-transitions | whether to disable animations        | ^[boolean]                                               | false   |
+| hit                 | whether Tag has a highlighted border | ^[boolean]                                               | false   |
+| color               | background color of the Tag          | ^[string]                                                | —       |
+| size                | size of Tag                          | ^[enum]`'large' \| 'default' \| 'small'`                 | default |
+| effect              | theme of Tag                         | ^[enum]`'dark' \| 'light' \| 'plain'`                    | light   |
+| round               | whether Tag is rounded               | ^[boolean]                                               | false   |
 
-| Name  | Description                  | Parameters |
-| ----- | ---------------------------- | ---------- |
-| click | triggers when Tag is clicked | —          |
-| close | triggers when Tag is removed | —          |
+### Tag Events
 
-## Slots
+| Name  | Description                  | Type                                                                |
+| ----- | ---------------------------- | ------------------------------------------------------------------- |
+| click | triggers when Tag is clicked | `(evt: MouseEvent)` => ^[Function]`(evt: MouseEvent) => boolean`          |
+| close | triggers when Tag is removed | `(evt: MouseEvent)` => ^[Function]`(evt: MouseEvent) => boolean`          |
+
+### Tag Slots
 
 | Name | Description               |
 | ---- | ------------------------- |
 | —    | customize default content |
 
-## CheckTag Attributes
 
-| Name    | Description | Type    | Accepted Values | Default |
-| ------- | ----------- | ------- | --------------- | ------- |
-| checked | is checked  | boolean | true/false      | —       |
+## CheckTag API
+
+### CheckTag Attributes
+
+| Name    | Description | Type       | Default |
+| ------- | ----------- | ---------- | ------- |
+| checked | is checked  | ^[boolean] | —       |
 
-## CheckTag Events
+### CheckTag Events
 
-| Name   | Description                        | Parameters |
-| ------ | ---------------------------------- | ---------- |
-| change | triggers when Check Tag is clicked | checked    |
+| Name   | Description                        | Type                                                            |
+| ------ | ---------------------------------- | --------------------------------------------------------------- |
+| change | triggers when Check Tag is clicked | `(value: boolean)` => ^[Function]`(value: boolean) => boolean`    |
```

Comment:
| change | triggers when Check Tag is clicked | ^[Function]`(value: boolean) => void` |